<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Три в ряд</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="scoreLabel">
      <property name="text">
       <string>Очки: 0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGridLayout" name="gridLayout">
     </widget>
    </item>
    <item>
     <widget class="QHBoxLayout" name="controlsLayout">
      <item>
       <widget class="QPushButton" name="newGameButton">
        <property name="text">
         <string>Новая игра</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveGameButton">
        <property name="text">
         <string>Сохранить игру</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadGameButton">
        <property name="text">
         <string>Загрузить игру</string>
        </property>
       </widget>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
